src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -1184,7 +1220,7 @@ private TexlNode ParseExprChain(TexlNode node, ITexlSource leftTrivia)
                 sourceList.Add(new TokenSource(delimiter));
                 sourceList.Add(ParseTrivia());
 
-                if (_curs.TidCur == TokKind.Eof || _curs.TidCur == TokKind.Comma || _curs.TidCur == TokKind.ParenClose)
+                if (_curs.TidCur == TokKind.Eof || _curs.TidCur == TokKind.Comma || _curs.TidCur == TokKind.ParenClose || _curs.TidCur == TokKind.CurlyClose)
```

Comment:
If we're not parsing udfs, does an expression `Foo; }; Bar` parse differently? It'd have been erroneous, so that's safe to change, but double checking.

Reply:
Yes